.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,46 @@
+name: Bug report
+description: File a bug report to help us improve
+title: "[BUG]"
+labels: ["bug", "🔍 needs-triage"]
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report! Contributing directly to this project is always encouraged. More details can be found in the [Contributing Guide](../../../CONTRIBUTING.md).
```

Comment:
The link here should just use 2 sets of '..'.  The link doesn't work (test is here: https://github.com/Azure/template-analyzer/blob/c8f5b84e0bd1e0fc62f2dba175b6b6624ed0daed/.github/ISSUE_TEMPLATE/bug_report.yml)